app/jobs/reports/month_helper.rb

```diff
@@ -8,19 +8,24 @@ module MonthHelper
     # @example
     #   months(Date.new(2021, 3, 15)..Date.new(2021, 5, 15))
     #   => [
-    #     Date.new(2021, 3, 15)..Date.new(2021, 3, 31),
-    #     Date.new(2021, 4, 1)..Date.new(2021, 4, 30),
-    #     Date.new(2021, 5, 1)..Date.new(2021, 5, 15),
+    #     Time.new(2021, 3, 15, 0, 0, 0)..Time.new(2021, 3, 31, 23, 59, 59),
+    #     Time.new(2021, 4, 1, 0, 0, 0)..Time.new(2021, 4, 30, 23, 59, 59),
+    #     Time.new(2021, 5, 1, 0, 0, 0)..Time.new(2021, 5, 14, 23, 59, 59),
     #   ]
     # @param [Range<Date>] date_range
-    # @return [Array<Range<Date>>]
+    # @return [Array<Range<Time>>]
```

Comment:
technically these are `ActiveSupport::TimeWithZone` ... but that was a lot more to write, and that's a subclass, so I felt like the gist was correct here